src/registry/email/ValueDisplay.scss

```diff
@@ -0,0 +1,3 @@
+@use '@utrecht/components/ordered-list';
+// TODO: the react components need to be updated, url-data vs. url-value
```

Comment:
this requires coordination with the SDK repo, and we're not going to do this so close to a stable release :)